libraries/api/discussion_helper.cpp

```diff
@@ -204,10 +209,20 @@ namespace golos { namespace api {
     }
 //
 // create_discussion
+    discussion discussion_helper::impl::create_discussion(const std::string& author) const {
+        auto dis = discussion();
+        golos::plugins::follow::fill_account_reputation(database_, author, dis.author_reputation);
```

Comment:
The discussion_helper receives pointer to fill_account_account_reputation in constructor.
Replace this with `fill_reputation_(...)`